lib/express/websocket.js

```diff
@@ -0,0 +1,65 @@
+import _ from 'lodash';
+import url from 'url';
+
+
+const WEBSOCKET_SERVERS = {};
+let IS_UPGRADE_LISTENER_ASSIGNED = false;
+const PATHNAME_PREFIX = '/ws';
+
+
+/**
+ * Adds websocket handler to express server instance.
+ *
+ * @param {Object} server - An instance of express HTTP server.
+ * @param {string} handlerPathname - Web socket endpoint path starting with
+ * single slash charcter. `/wd/ws` prefix is automatically added to the
+ * actual endpoint.
+ * @param {Object} handlerServer - WebSocket server instance. See
+ * https://github.com/websockets/ws/pull/885 for more details
+ * on how to configure the handler properly.
+ * @param {?string} pathnamePrefix ['/ws'] - Pathname prefix.
+ */
+function addWebsocketHandler (server, handlerPathname, handlerServer, pathnamePrefix = PATHNAME_PREFIX) {
```

Comment:
Isn't `WebSocket` with a capital `S`? It would be good to reflect that in the function names.

Reply:
it is not a problem at all to change that